antora.yml

```
@@ -106,7 +116,7 @@ asciidoc:
     open_api_spec_url: https://spec.openapis.org/oas/v3.1.0.html
     open_api_swagger_spec_url: https://swagger.io/docs/specification
     quarkus_openapi_gen_url: https://github.com/quarkiverse/quarkus-openapi-generator
-    kn_workflow_plugin_releases_url: https://developers.redhat.com/content-gateway/rest/mirror/pub/cgw/serverless-logic/1.33.0/
+    kn_workflow_plugin_releases_url: https://developers.redhat.com/content-gateway/rest/mirror/pub/cgw/serverless-logic/{product_version_short}.0/
```

What about? just a nitpick

```suggestion
    kn_workflow_plugin_releases_url: https://developers.redhat.com/content-gateway/rest/mirror/pub/cgw/serverless-logic/{product_version_long}/
```